server/stycoon/command_trade.go

```diff
@@ -26,6 +26,10 @@ func (game *Game) processTrade(id int64, playerOwnedShips map[int64]struct{}, pl
 	_, shipOwnedByPlayer := playerOwnedShips[id]
 	_, targetIsPlanet := planetIDs[id]
 
+	if !shipOwnedByPlayer && !targetIsPlanet {
+		return fmt.Errorf("trading with another players ship is forbidden")
```

Comment:
"trading with wreck or another player ship is forbidden"

Reply:
that is, trading with ones own ships is allowed